contracts/payment/Escrow.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.4.23;
+
+import "../math/SafeMath.sol";
+
+
+/**
+ * @title Escrow
+ * @dev Base escrow contract, holds funds destinated to a payee until they
+ * withdraw them.
+ */
+contract Escrow {
```

Comment:
We should add `Deposited` and `Withdrawn` events. (I'm open to other names.)

Reply:
This would make the `Refunded` event a bit redundant (since it will also emit an identical `Withdrawn`): do we want to remove it?